app/templates/preprints/view.hbs

```diff
@@ -3,74 +3,51 @@
         <div class="m-b-md p-md osf-box m-t-lg"> {{!OSF BOX DIV}}
             <div class="row" id="header"> {{!HEADER ROW}}
                 <h1>{{model.preprint.title}}</h1>
-                <h5>{{model.preprint.authors}}</h5>
-                <h5>Added on {{moment-format model.preprint.date}}</h5>
             </div> {{!END HEADER ROW}}
-            <hr color="#C7FFC7">
-            <div class="row"> {{!CONTENT ROW}}
-                <div id="mfr" class="col-md-7"> {{!MFR COL DIV}}
-                    <div> {{!IFRAME DIV}}
-                        <iframe src={{concat model.renderUrl '?url=' model.downloadLink}} display="block"
-                                width=650 height=700></iframe>
-                    </div> {{!END IFRAME DIV}}
-                    {{!TODO: EMBER OSF MFR }}
-
-                </div> {{!END MFR COL DIV}}
+            <div class="row p-l-md"> {{!PREPRINT INFO ROW}}
+                <div class="col-md-10">
+                    <h5>{{model.preprint.authors}}</h5>
+                    <h6>Added on {{moment-format model.preprint.date}}</h6>
+                </div>
+                <div class="col-md-2">
+                    <h6 class="m-t-lg"><span>Provider: {{#link-to "index"}}SocArXiv{{/link-to}} </span><img src="preprints/assets/img/logo_socarxiv.jpg" width="30"></h6>
```

Comment:
If you happen to see any weirdness with image display after running inside the OSF, this may be helpful- rather than hardcoding images, we're using styles throughout.

(ember uses this to handle rootURL behavior)

https://github.com/CenterForOpenScience/ember-preprints/commit/5685f833828c262669a71f5b4b4b8363457f8f28#diff-2de80de59c74fd0a521d4a0dc3531cddR734